squirrel/plugin.py

```diff
@@ -18,30 +18,77 @@ def load_module():
         return importlib.import_module(f'squirrel.plugins.{project_type}')
 
     @staticmethod
-    def get_files(path):
-        find_output = subprocess.run(
-            f'find {path} -type f -not -path "*/[@.]*"',
-            shell=True,
-            capture_output=True,
-            text=True)
-        return find_output.stdout.strip().split('\n')
+    def get_files(wd, ignores):
+        """Function to find all non-ignored files i project directory"""
+        ignore = ignores.get('ignore')
+        project_files = set()
+        for file in iglob('**/*', recursive=True):
+            if path.isfile(file):
+                file_path = path.join(wd, file)
+                project_files.add(file_path)
+        project_files.difference_update(ignore)
+        return project_files
+
+    @staticmethod
+    def import_ignores(wd, file, logger):
+        """Function to read ignore file and store extensions, dir and files
+         to a dictionary"""
+        ignores = {
+            'dir': set(),
+            'file': [],
+            'ignore': set(),
+            }
+        try:
+            with open(file, 'r') as file:
+                tmp_ignore = []
+                tmp_dir = []
+                for line in file.readlines():
+                    add_line = line.strip()
+                    if add_line.startswith('#') or add_line == '':
+                        continue
+                    elif add_line.endswith('/'):
+                        # Add ignored folders and sub folders
+                        # to be passed to Handler()
+                        tmp_dir.extend(
+                            glob(f'{add_line}/**/', recursive=True,))
```

Comment:
I think that this will also have the same bug as in the my other comment.

Reply:
I added glob here to ignore sub directories inside a ignored dir. But I'll change this one too.

Reply:
Ohh then we should keep it that way. It's the more logical behaviour.